packages/edit-site/src/components/page-actions/index.js

```diff
@@ -10,17 +10,11 @@ import { moreVertical } from '@wordpress/icons';
  */
 import TrashPageMenuItem from './trash-page-menu-item';
 
-export default function PageActions( {
-	postId,
-	className,
-	toggleProps,
-	onRemove,
-} ) {
+export default function PageActions( { postId, toggleProps, onRemove } ) {
 	return (
 		<DropdownMenu
 			icon={ moreVertical }
 			label={ __( 'Actions' ) }
-			className={ className }
```

Comment:
It seems `PageActions` with the `className` property was introduced a while ago at 375b16fb9b0383958e9f63c88ebafe3dcb06dd76 Probably best to leave the component as it is?